package.json

```diff
@@ -5,54 +5,29 @@
-    "test": "npm run test:lint && npm run dataValidate && npm run test:dependencies && npm run test:unit && npm run test:ampValidate:ci",
-    "test:accessibility": "npm run killApp && npm run build && run-p --race start bbcA11y",
-    "test:ampValidate": "npm run killApp && npm run build -- --hide-modules --colors && run-p --race start amp:validate",
-    "test:ampValidate:ci": "npm run build -- --hide-modules --colors && run-p --race start amp:validate",
-    "test:chromatic": "chromatic test run  --build-script-name build:storybook || true",
+    "test": "npm run test:lint && npm run test:dependencies && npm run test:unit && npm run build -- --hide-modules --colors && run-p --race start amp:validate",
+    "test:chromatic": "chromatic test run --build-script-name build:storybook || true",
+    "test:ci": "npm run start & npm install --no-save cypress@latest bbc-a11y@latest lighthouse@latest npm-run-all@latest puppeteer@latest && run-p cypress test:puppeteer && run-p bbcA11y lighthouse",
     "test:dependencies": "npx dependency-check@3.4.1 ./package.json",
-    "test:prod:ci": "npm run start & npm install --no-save cypress@latest bbc-a11y@latest lighthouse@latest npm-run-all@latest puppeteer@latest && run-p cypress test:puppeteer && run-p bbcA11y lighthouse",
-    "test:prod:travis": "CYPRESS_SKIP_EU=true npm run test:prod:ci",
-    "test:e2e": "npm run killApp && npm run build && run-p --race start cypress",
-    "test:e2e:local_env": "CYPRESS_APP_ENV=local npm run test:e2e",
-    "test:e2e:test_env": "CYPRESS_APP_ENV=test npm run cypress",
-    "test:e2e:live_env": "CYPRESS_APP_ENV=live npm run cypress",
-    "test:e2e:interactive": "npm run killApp && npm run build && run-p --race start cypress:interactive",
-    "lighthouse": "npm run lighthouse:article && npm run lighthouse:frontpage",
-    "lighthouse:article": "lighthouse http://localhost.bbc.com:7080/news/articles/c6v11qzyv8po --chrome-flags=\"--no-sandbox --headless --disable-gpu\" --output json --output html --output-path simorgh --config-path scripts/lighthouseConfig.js && node scripts/lighthouseBudget.js run",
-    "lighthouse:frontpage": "lighthouse http://localhost.bbc.com:7080/igbo --chrome-flags=\"--no-sandbox --headless --disable-gpu\" --output json --output html --output-path simorgh --config-path scripts/lighthouseConfig.js && node scripts/lighthouseBudget.js run",
     "test:lint": "eslint --ext .js,jsx,json ./src ./data ./cypress ./.storybook ./webpack** && swagger-cli validate data/schema.yaml",
     "test:lint:css": "stylelint 'src/**/*.js' 'src/**/*.jsx'",
     "test:puppeteer": "jest --env=jsdom --colors ./puppeteer",
-    "test:size": "./scripts/bundleSize.sh",
     "test:unit": "jest --env=jsdom --coverage --colors --testPathIgnorePatterns=\"puppeteer\"",
-    "test:unit:watch": "jest --env=jsdom --coverage --colors --testPathIgnorePatterns=\"puppeteer\" --watch",
-    "snyk": "snyk test",
-    "updateSnapshots": "npm run test:unit -- -u",
```

Comment:
i use this command :( 

i guess i can use `npm run test:unit -- -u`

Reply:
Kill it with fire, please and thank you.